Update how-to-compliant-network.md (#1204)
MicrosoftDocs/entra-docs: docs/global-secure-access/how-to-compliant-network.md

1. Select the **Create** button to create to enable your policy.

> [!NOTE]

> > You can use Global Secure Access along with a Conditional Access policies requiring a Compliant Network for All Resources.

Reviewer: ```suggestion
> Use Global Secure Access along with Conditional Access policies that require a Compliant Network for All Resources.
```

> > Global Secure Access resources are automatically excluded from the Conditional Access policy when Compliant Network is enabled in the policy. There's no explicit resource exclusion required. These automatic exclusions are needed to ensure that the Global Secure Access client is not blocked from accessing the required resources Global Secure Access needs: 

Reviewer: ```suggestion
> Global Secure Access resources are automatically excluded from the Conditional Access policy when *Compliant Network* is enabled in the policy. There's no explicit resource exclusion required. You need these automatic exclusions to ensure that the Global Secure Access client is not blocked from accessing the required Global Secure Access resources. The resources Global Secure Access needs are: 
```
